public/app/percona/shared/core/reducers/dbaas/addDBCluster/addDBCluster.ts

```diff
@@ -103,8 +103,8 @@ export const addDbClusterAction = createAsyncThunk(
         memory,
         disk,
         databaseImage: databaseVersion?.value,
-        expose,
-        internetFacing,
+        expose: !!expose,
+        internetFacing: expose && !!internetFacing,
```

Comment:
I think this may still result in undefined/non-boolean value:
`undefined && false` results in `undefined`.

either add `!!` before expose or do `!!(expose && internetFacing)`

Reply:
fixed